userguide/storage.rst

```diff
@@ -1280,6 +1289,51 @@ changed, and destroying a zvol requires confirmation.
 Managing Encrypted Volumes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+%brand% automatically generates a randomized **encryption key** 
+whenever a new encrypted volume is created. The data "at rest" (on disk)
+on an encrypted volume is **always encrypted**, but access to the decrypted
+data on a live (running) system can be customized for each volume.
```

Comment:
This is confusing.  It seems to be an introduction to when the encryption keys can be entered, but the word "access" can be read to mean several other things.

Reply:
Will fix by merging into para below bullets, which already mentions that topic so it also avoids possible duplication.